api/go.mod

```diff
@@ -8,14 +8,14 @@ require (
 	github.com/golang/protobuf v1.5.2
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	google.golang.org/genproto v0.0.0-20210602131652-f16073e35f0c
-	google.golang.org/grpc v1.41.0
+	google.golang.org/grpc v1.47.0
 )
 
 require (
-	golang.org/x/net v0.17.0 // indirect
-	golang.org/x/sys v0.13.0 // indirect
-	golang.org/x/text v0.13.0 // indirect
-	google.golang.org/protobuf v1.26.0 // indirect
+	golang.org/x/net v0.0.0-20210405180319-a5a99cb37ef4 // indirect
```

Comment:
This isn't correct. You reverted the `golang.org/x/net` version.

Reply:
It's a result of `go mod tidy` after `git rebase` that reverted this std lib version. 

```
dev-dsk-chaochn-2c-a26acd76 % go mod graph | grep "golang.org/x/tools@v0.1.2"
google.golang.org/genproto@v0.0.0-20210602131652-f16073e35f0c golang.org/x/tools@v0.1.2

dev-dsk-chaochn-2c-a26acd76 % go mod graph | grep "golang.org/x/net"
golang.org/x/tools@v0.1.2 golang.org/x/net@v0.0.0-20210405180319-a5a99cb37ef4
```

In conclusion, it was reverted because of `google.golang.org/genproto` was stick with old version.

Reply:
I can manually bump up indirect dependency though.